### PR TITLE
Add runner script

### PR DESCRIPTION
**Pull request by johnnv1**
The goal here is to allow run multiple performance tests with a simple `config.*.py` file and adding it to the config YAML file.

### CODE REVIEW

**Comment by edgarriba** [comment]
by default it runs morphology -- how do we run other, can you add short usage docs ?

**Comment by johnnv1** [comment]
oops, sorry, i forgot, i will add it now

File: dynamo/runner.py [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+                        desc = f'{_opt_name}{operator.split("_")[0]}_{device}'
+                        stmt = f'{operator}(input, **kwargs)'
+                        setup = f'from {import_from} import {operator}'
+
+                        bench_out = benchmark.Timer(
+                            stmt=stmt,
+                            setup=setup,
+                            globals={'input': x, 'kwargs': kwargs},
+                            num_threads=num_threads,
+                            label=module_name,
```
Comment (johnnv1) [inline dynamo/runner.py:284]: TODO: `env=` and have an appropriate colect_env for our case  -- that can collect information from the system, that also works on hardware that we care about, such as a jetson

- https://github.com/rbonghi/jetson_stats/tree/master
- https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py

We need to collect the version/commit of which kornia was used too

**Comment by edgarriba** [comment]
How can we easily compare an existing implementation over an improvement or new one? Like in https://github.com/kornia/kornia/pull/2071

**Comment by johnnv1** [comment]
right now, we need to do manually some things... like for this case which have a new implementation, we need to:
- create a bench script like `config/color/multiple.py` or add to this one the new cases
- add this to the config - `bench_config.yaml`
- run the runner 


For updates in some implementation I use the `env` case, so first bench against the master, change the branch e test against the pr -- I use this approach [here](https://github.com/kornia/kornia/pull/2059) with this other [script](https://github.com/johnnv1/kornia/blob/d33851ad8a446aa189ee34738f8fd994cf76a1b3/performance-check.py)

**Comment by johnnv1** [comment]
I have already put it on the TODO list of our doc

**Pull request merged**